scripts/microbe_stage/organelle.as

```diff
@@ -205,6 +208,8 @@ class Organelle{
 
     array<OrganelleComponentFactory@> components;
     private dictionary hexes;
+    int cachedHexCount = 0;
```

Comment:
This should be made private and accessed only with a const method `getHexCount` (see my other new comment)